shell/postman-openapi-refresh.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import json
+import logging
+import sys
+
+from dataclasses import dataclass
+
+
+@dataclass
+class Section:
+    name: str
+    resources: dict
+
+
+# Division of the resources by Section
+# When resource value is set to None, tag will be automatically generated
+# from path prefix replacing _ with spaces and capitalising each word
+SECTIONS = (
+    Section('Core',
```

Comment:
What will happen to a new resource that is not added in none of these lists?

Reply:
Resource will be skipped and a warning will be printed (line 64). 
Anyway that default behaviour can be changed if needed.